procrastinate/shell.py

```diff
@@ -96,9 +97,9 @@ def do_retry(self, arg):
 
         Example: retry 2
         """
-        print_job(self.admin.set_job_status(arg, status="todo"))
+        print_job(self.admin.set_job_status(arg, status="todo"))  # type: ignore
```

Comment:
Why do you need to ignore typing here?

Reply:
mypy undestands `set_job_status_async` but not `set_job_status`.

https://github.com/peopledoc/procrastinate/issues/101